crates/bevy_window/Cargo.toml

```diff
@@ -20,6 +20,7 @@ bevy_input = { path = "../bevy_input", version = "0.9.0-dev" }
 raw-window-handle = "0.4.2"
 
 # other
+serde = { version = "1.0", features = ["derive"] }
```

Comment:
you've put use of serve behind a `serialize ` feature, but you didn't put this crate as optional and didn't add the `serialize` feature

Reply:
I did what I thought I saw `bevy_input` doing.  I'll change it to add those things.

Reply:
A little scary that the ci tests all passed.

Reply:
You've added code that is always disabled and no tests on it. It's impossible for CI to detect that 😄

Reply:
I understand.